src/test/java/core/di/factory/ComponentScanTest.java

```diff
@@ -0,0 +1,49 @@
+package core.di.factory;
+
+import java.util.Set;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.reflections.Reflections;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import core.annotation.Repository;
+import core.annotation.Service;
+import core.annotation.web.Controller;
+
+public class ComponentScanTest {
+	private static final Logger logger = LoggerFactory.getLogger(ComponentScanTest.class);
+
+	private Reflections reflections;
+
+	@BeforeEach
+	public void setUp() {
+		reflections = new Reflections("core.di.factory.example");
+	}
+
+	@Test
+	@DisplayName("@Controller Annotation 클래스 찾기")
+	public void findControllerAnnotation() {
+		Set<Class<?>> typesAnnotation = reflections.getTypesAnnotatedWith(Controller.class);
+
+		typesAnnotation.stream().forEach(annotation -> logger.debug("Class Name : {}", annotation.getName()));
```

Comment:
```suggestion
		typesAnnotation.forEach(annotation -> logger.debug("Class Name : {}", annotation.getName()));
```
Collection 에서 forEach 또한 Consumer 를 활용하고 있어서 위와 같이 좀 더 간단하게 될 수 있네요~